cirq/ops/common_gates.py

```diff
@@ -81,6 +81,12 @@ def _apply_unitary_(self, args: protocols.ApplyUnitaryArgs
             args.available_buffer *= p
         return args.available_buffer
 
+    def as_rotation(self):
```

Comment:
Need better names. The expression `(X**0.5).as_rotation()` is a bit of a mystery, I think. It's already a rotation..

I think what you want is `in_su2` or `with_non_imaginary_global_phase` or something like that. Describe at a lower level what it means for it to be "a rotation" and put that in the name.

Reply:
I like in_su2 since that describes the exact purpose why someone would want to use this transformation. I'll go with that.